src/test/java/com/wikia/webdriver/testcases/flagTests/flagTest.java

```diff
@@ -0,0 +1,47 @@
+package com.wikia.webdriver.testcases.flagtests;
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.common.properties.Credentials;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+/**
+ * Created by Mariusz on 2015-06-28.
+ */
+public class FlagTest extends NewTestTemplate {
```

Comment:
Always keep class name the same as file name
your class name is FlagTests and file name starts with lower case letter
it is complitation error. What IDE are you using ? your IDE should inform you of such errors